beetsplug/bpd/__init__.py

```diff
@@ -542,12 +614,17 @@ def cmd_seekid(self, conn, track_id, pos):
         index = self._id_to_index(track_id)
         return self.cmd_seek(conn, index, pos)
 
+    # Debugging/testing commands that are not part of the MPD protocol.
+
     def cmd_profile(self, conn):
         """Memory profiling for debugging."""
         from guppy import hpy
         heap = hpy().heap()
         print(heap)
 
+    def cmd_crash_TypeError(self, conn):
+        'a' + 2
```

Comment:
This looks a little silly, but I understand why it's there. Maybe it could use a quick comment?

Reply:
Done!